src/main/scala/Day1/Main.scala

```diff
@@ -130,18 +131,28 @@ object Main extends App {
     } else if (chooseOption == 5) {
```

Comment:
else if (chooseOption == 5) body is the same as  if (chooseOption == 1)
you could just put it inside the if and do an OR check